common/persistence/sql/sqlQueue.go

```diff
@@ -26,15 +26,14 @@ import (
 	"database/sql"
 
 	workflow "github.com/uber/cadence/.gen/go/shared"
-	"github.com/uber/cadence/common"
 	"github.com/uber/cadence/common/log"
 	"github.com/uber/cadence/common/persistence"
 	"github.com/uber/cadence/common/persistence/sql/sqlplugin"
 )
 
 type (
 	sqlQueue struct {
-		queueType common.QueueType
+		queueType persistence.QueueType
```

Comment:
consider doing renaming in a separate PR in the future. Its easier to review as well as to revert etc

Reply:
Sure. Will follow this in the future.